CONTRIBUTING.md

```diff
@@ -45,18 +45,11 @@ during their normal contribution hours.
 
 ## Development
 
-To quickly get up and running, you can use the `scripts/eachdist.py` tool that
-ships with this project. First create a virtualenv and activate it.
-Then run `python scripts/eachdist.py develop` to install all required packages
-as well as the project's packages themselves (in `--editable` mode).
 
-Further, you'll want to clone the Contrib repo locally to resolve paths needed
+First, clone the Contrib repo locally to resolve paths needed
 to run tests. `git clone git@github.com:open-telemetry/opentelemetry-python-contrib.git opentelemetry-python-contrib`.
 
-You can then run `scripts/eachdist.py test` to test everything or
```

Comment:
Is it possible to run tox without linting (only tests)? For me, the pylint step takes probably longer than everything else together.

Reply:
Yes, `tox -e py39-test-core-api`, for example.

Reply:
OK, but I guess there is no way to run all tests without either listing them one by one or also running lint, etc at the same time. Not a must-have feature though.